pjsip/include/pjsua2/endpoint.hpp

```diff
@@ -1162,11 +1162,24 @@ struct EpConfig : public PersistentObject
 /* This represents posted job */
 struct PendingJob
 {
+    PendingJob():autoDelete(true){};
+
+    PendingJob(bool autoDel):autoDelete(autoDel) {};
```

Comment:
- Instead of having two constructors, why not just one with default param, e.g: `PendingJob(bool autoDel=true)`?
- IMO it is better to have the docs about maintaining reference to object `PendingJob` for Python/Java, manual delete, destructor called from PJLIB thread, and mini sample code (or link to sample) here?